kuma/users/apps.py

```diff
@@ -24,3 +30,8 @@ def ready(self):
         if settings.STRIPE_SECRET_KEY:
             stripe.api_key = settings.STRIPE_SECRET_KEY
             stripe.max_network_retries = settings.STRIPE_MAX_NETWORK_RETRIES
+
+            try:
+                create_missing_stripe_webhook()
```

Comment:
This will trigger an error when running tests:
```
stripe.error.AuthenticationError: Invalid API Key provided: te*ting
```
The solution isn't swallow errors but to avoid them in the first place.

Reply:
So I think it should become just:
```python
if settings.STRIPE_PUBLIC_KEY != "testing":
    create_missing_stripe_webhook()
```

Reply:
Fair enough. Thank you (and also for supplying a fix)!